### PR TITLE
Adding new Scala and Java benchmarks

### PR DESCRIPTION
**Pull request by marcospereira**
# Dependencies

This depends on #5 and #6.

## Purpose

This will activate new Java and Scala benchmarks. Two new Lua scripts were added to support the new benchmarks. Apparently, wrk is capable of handling more than one script, so we can have something like `wrk ... -s wrk_report.lua -s wrk_post.lua ...`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +44,14 @@ Test an action that returns a short HTML page generated from a template that tak
 
 Test an action that returns a short HTML page generated from a template that takes an implicit language.
 
+----------------
+
+All the tests above run for three distinct configurations:
+
+1. Using Netty server
+2. Using [Akka HTTP](https://github.com/akka/akka-http) Server with default filters
+3. Using [Akka HTTP](https://github.com/akka/akka-http) Server **without** default filtersca
```
Comment (gmethvin) [inline README.md]: typo at the end

**Comment by marcospereira** [comment]
@richdougherty this is ready to be reviewed.

Keep in mind that we first need to nail down #5 and #6.

Also, I have noticed that we don't have all possible benchmarks turned on. For example, we don't have `scala-minimal-template-simple`, but only `scala-di-template-simple`. That would explain why some of the graphs are empty.

Finally, I've made the changes only to `apps-master-10`. Please let me know if we need to also port these to `apps-master-8`.

**Comment by richdougherty** [comment]
The `minimal` apps have the default filters disabled, including the CSRF filter. The form upload tests shouldn't check for a CSRF token.

For the other apps where the CSRF filter is still enabled (`di`, `netty`, etc) should we set up the form tests so we supply a real CSRF token?, i.e. have wrk supply a cookie value.

**Comment by marcospereira** [comment]
> For the other apps where the CSRF filter is still enabled (di, netty, etc) should we set up the form tests so we supply a real CSRF token?, i.e. have wrk supply a cookie value.

It is possible, I think. But we should maybe do that in another PR. :-)

**Comment by richdougherty** [comment]
Another PR sounds fine to me. :)

File: README.md (hunk 1 above)
Comment (marcospereira) [reply README.md]: Fixed. :-)

**Pull request merged**